ocs_ci/deployment/helpers/prechecks.py

```diff
@@ -112,6 +112,10 @@ def template_check(self):
             TemplateNotFound: If template not found in Datacenter.
 
         """
+        # skip the template check for vSphere IPI
+        if self.platform == VSPHERE_PLATFORM and self.deployment_type == "ipi":
```

Comment:
Just small note, since it is part of `VSpherePreChecks` class, is it required to check the also `platform` here?

Also it might be worth to mention it in the docstring, that this check is applicable only for `UPI` installation.

Reply:
done